lib/cisco_node_utils/cmd_ref/acl_v4.yaml

```diff
@@ -0,0 +1,25 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
+_template:
+  config_get: 'show run aclmgr'
+  config_get_token: '/^ip access-list <acl_name>$/'
+  config_set: 'ip access-list <acl_name>'
+
+ace:
+  config_get_token_append: '/\s+(<seqno>)\s+(permit|deny)\s+(sctp|tcp|ip|udp|pim|icmp|igmp|gre|nos|pcp|ospf|eigrp|ahp|esp|\d+)\s+(any|host \S+|\S+\/\d+|\S+ [:\.0-9a-fA-F]+)\s+(eq \d+|neq \d+|lt \d+|gt \d+|range \d+ \d+)*\s*(any|host \S+|\S+\/\d+|\S+ [:\.0-9a-fA-F]+)\s*(eq \d+|neq \d+|lt \d+|gt \d+|range \d+ \d+)*( [a-zA-Z0-9\/ ]*)*$'
```

Comment:
My suggestion would be to use a simpler regexp like the following.

`/<seqno> (/S+) (/S+) (.*)/` and then do all your regexp post processing the the object methods.  The reason I chose (.*) for everything the follows the protocol is because there are many cases you will have to match and it would be simpler to match in a method rather then try and match everything in the yaml.